packages/js/product-editor/src/components/variations-table/use-variations/use-variations.ts

```diff
@@ -285,6 +285,7 @@ export function useVariations( { productId }: UseVariationsProps ) {
 
 		return updateProductVariation(
 			{ product_id: productId, id: variationId },
+			// @ts-expect-error not sure that it is necessary
 			variation
```

Comment:
@louwie17 if you have the bandwidth, could you double-check if this parameter is necessary?

Reply:
Yeah, that is required!
See: https://github.com/woocommerce/woocommerce/blob/37fc4258fe0a9e26b0be4d2d3d75e075ecd31f7e/packages/js/data/src/crud/actions.ts#L233-L236

Reply:
I believe this line:
https://github.com/woocommerce/woocommerce/blob/9137666efdf914e3a6252fef6e84612b3ab9675c/packages/js/data/src/crud/types.ts#L63

Should be:
`update: ( idQuery: IdQuery, query: Partial< ItemType > ) => Item;`

Reply:
Yeah, it makes sense! Thanks! 🙇